features/cucumber_events.feature

```diff
@@ -1,6 +1,6 @@
 Feature: Cucumber events
 
-  Scenario: BeforeFeatures and AfterFeatures only fired once
+  Scenario: BeforeFeatures and AfterFeatures are fired for every scenario
```

Comment:
@peter-reperger is it possible to keep the previous behaviour where BeforeFeatures and AfterFeatures are only fired once and not for every scenario?  I think it removes their usefulness having then fire for every scenario as you can't use them for setup and teardown logic that only needs/wants to be fired at the very start and very end of the test run.